spec/spec_helper.rb

```diff
@@ -1,3 +1,4 @@
+require "active_support"
 require "active_support/testing/time_helpers"
```

Comment:
🤔  if we pulling in `active_support` above we might not need to explicitly require this?

Reply:
it completely broke without it I've no idea why it has to be so explicit so keeping it in